### ISSUE TITLE
Spacer block: unlabeled Range and Input controls 

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#63800]
### Description

The Spacer block can be used to add vertical or horizontal spacing.

In both cases, two settings controls are unlabeled. One more case of completely unlabeled controls only because the `label` prop is not required or used incorrectly. Cc @WordPress/gutenberg-components @joedolson 

Vertical spacing:
The range control highlighted in the screenshot below is unlabaled:

![Screenshot 2024-07-22 at 12 51 45](https://github.com/user-attachments/assets/0ba62dca-373d-4b98-b0f4-3f7516500213)

Horizontal spacing:
The input field highlighted in the screenshot below is unlabaled:

![Screenshot 2024-07-22 at 12 51 39](https://github.com/user-attachments/assets/5cb69420-6105-4b6f-aedd-ef2565b1913e)

### Step-by-step reproduction instructions

- Add a Row with a paragraph, a spacer block and another paragraph
- Select the spacer.
- Inspect the width input control when the width is set to 'fixed'. 
- Observe the input field is not labeled.
- Add a normal Spacer block and select it.
- Inspect the width range control.
- Observe it's not labeled.

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes
